libs/shared/pem-jwk/src/lib/pem-jwk.spec.ts

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { pem2jwk, jwk2pem } from './pem-jwk';
+
+describe('pem2jwk', () => {
+  it('converts private RSA key', () => {
+    expect(pem2jwk(privateRsaKey)).toMatchObject(privateRsaKeyJwk);
+  });
+
+  it('converts public RSA key', () => {
+    expect(pem2jwk(publicRsaKey)).toMatchObject(publicRsaKeyJwk);
+  });
+
+  it('converts public key', () => {
+    expect(pem2jwk(publicKey)).toMatchObject(publicKeyJwk);
+  });
+});
+
+describe('jwk2pem', () => {
+  it('converts private RSA JWK', () => {
+    expect(jwk2pem(privateRsaKeyJwk)).toBe(privateRsaKey);
+  });
+
+  it('converts public RSA JWK', () => {
+    expect(jwk2pem(publicRsaKeyJwk)).toBe(publicRsaKey);
+  });
+});
+
+describe('conversions match', () => {
+  it('should work with public keys', () => {
+    const publicJwk = pem2jwk(publicRsaKey);
+    expect(publicRsaKey).toBe(jwk2pem(publicJwk));
+  });
+
+  it('should work with private keys', () => {
+    const privateJwk = pem2jwk(privateRsaKey);
+    expect(privateRsaKey).toBe(jwk2pem(privateJwk));
+  });
+});
+
+const privateRsaKey = `
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
oh boy. brace for automated scanners to start filing sec tickets...

Reply:
Yea, I could base64 it to stop the false positives (presuming they don't base64 decode it).